wrappers/rulegraph/README.md

```diff
@@ -16,18 +16,22 @@ Params section:
   file or directory). Rules provided in this dictionary will be
   shown in blue and clickable in the ouptut SVG file.
 - **configname**: a config file required by the input Snakefile
+- **required_local_files**: a list of required file and directory next to
```

Comment:
a list of required file and directory next to the Snakefile that necessary to run the pipeline.
>
a list of files and directories next to the Snakefile that are required to run the pipeline.

Reply:
Ty !